v1-preparations/REMAINING_GA_OBJECTS.MD

```diff
@@ -36,7 +36,7 @@ Known issues lists open issues touching the given object. Note that some of thes
 | PACKAGES POLICY             |   ❌    | -                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | PASSWORD POLICY             |   ❌    | [#2213](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2213), [#2162](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2162)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| PIPE                        |   ❌    | [#2075](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2075), [#1781](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/1781), [#1707](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/1707), [#1478](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/1478), [#533](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/533)                                                                                                                                                                                                                                                                  |
+| PIPE                        |   ❌    | [#2075](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2075), [#1781](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/1781), [#1707](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/1707), [#1478](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/1478), [#533](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/533), [#2785](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2785)                                                                                                                                                                                                                                                                  |
```

Comment:
just FYI, all these issues were ordered in a desc order ;) but do not change it if there are no other comments